src/main/java/me/jellysquid/mods/sodium/client/gui/options/OptionImpact.java

```diff
@@ -17,7 +19,8 @@
         this.text = text;
     }
 
-    public String toDisplayString() {
-        return this.color + this.text;
+    @Override
+    public Text getText() {
+        return new TranslatableText(this.text).formatted(this.color);
```

Comment:
Might help with reducing allocations if you construct the text in constructor?

Reply:
I'm not sure if this is the case, as Sodium's options are built on Screen initialization. I may be wrong since I'm unsure.